SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.html

```diff
@@ -0,0 +1,49 @@
+<head>
```

Comment:
I think that this head elements are not necessary

Reply:
deleted